bundles/org.openhab.ui/web/README.md

```diff
@@ -0,0 +1,68 @@
+# openHAB
```

Comment:
this might want to have a better title

Reply:
This file was actually auto-generated when scaffolding the project, not sure if it's really useful since there would already be a README in the parent folder. Some information on how to start the development server and contribute etc. could appear in a separate CONTRIBUTING.md.